src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.TextWriter.cs

```diff
@@ -142,8 +142,15 @@ private bool CheckIfWritingJson(string currentContentType)
         internal sealed class ODataUtf8JsonTextWriter : TextWriter
         {
             private readonly ODataUtf8JsonWriter jsonWriter = null;
+            // Buffer used to store chars that were could not be encoded due to

```

Comment:
Since you decided to describe what the private fields do, it'd be better to use `/// <summary></summary>`. That way the intellisense feature display the description wherever the field is used